components/icons/images/tier1/ai.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="none" viewBox="0 0 18 18">
+  <path fill="#202122" d="M3 1a.75.75 0 0 1 .75.75 1.5 1.5 0 0 0 1.5 1.5.75.75 0 0 1 0 1.5 1.5 1.5 0 0 0-1.5 1.5.75.75 0 0 1-1.5 0 1.5 1.5 0 0 0-1.5-1.5.75.75 0 0 1 0-1.5 1.5 1.5 0 0 0 1.5-1.5A.75.75 0 0 1 3 1ZM11.5 2.5a1 1 0 0 1 .928.629c.82 2.05 1.138 2.79 1.645 3.298.508.507 1.248.825 3.298 1.645a1 1 0 0 1 0 1.856c-2.05.82-2.79 1.138-3.298 1.645-.507.508-.825 1.248-1.645 3.298a1 1 0 0 1-1.857 0c-.82-2.05-1.137-2.79-1.644-3.298-.507-.507-1.248-.825-3.298-1.645a1 1 0 0 1 0-1.856c2.05-.82 2.79-1.138 3.298-1.645.507-.507.825-1.248 1.645-3.298A1 1 0 0 1 11.5 2.5ZM4.781 13.281a.781.781 0 0 0-1.562 0c0 .518-.42.938-.938.938a.781.781 0 1 0 0 1.562c.518 0 .938.42.938.938a.781.781 0 0 0 1.562 0c0-.518.42-.938.938-.938a.781.781 0 1 0 0-1.562.938.938 0 0 1-.938-.938Z"/>
```

Comment:
I think the fill on these should be tungsten (`#494c4e`)? Otherwise the ability to change the colour of the icons won't work since it's [specifically looking for that](https://github.com/BrightspaceUI/core/blob/main/components/icons/fix-svg.js#L5).

Reply:
Ah interesting, will fix!